src/assets/locales/de/main.json

```diff
@@ -833,7 +833,8 @@
         "resetCredentialError": "Something went wrong",
         "resetCredentialSuccess": "Credential reset was successful",
         "tabs": {
-          "all": "All",
+          "active": "Active",
+          "inactive": "Inactive",
```

Comment:
Please add german words instead of english

Reply:
@lavanya-bmw Added